WebRoot/js/ajax/dwt/widgets/DwtBaseDialog.js

```diff
@@ -242,7 +242,8 @@ function(loc) {
 	kbMgr.pushTabGroup(this._tabGroup);
 	kbMgr.pushDefaultHandler(this);
 
-	DwtShell.getShell().addListener(DwtEvent.CONTROL, this._resizeHdlr.bind(this));
+	this.shellResizeHandler = this._resizeHdlr.bind(this);
```

Comment:
@barrydegraaff , can we move `this.shellResizeHandler = this._resizeHdlr.bind(this);` definition to object's constructor function.

Thanks